test/templates/popover/index.njk

```diff
@@ -140,5 +148,22 @@ is simply a styling example.</p>
 {% endset %}
 {{ format(example) }}
 
+<br><br><br><br>
+
+<h2>Full width of parent with popper.js</h2>
+<p>NOTE: This will not be functional as the popper.js library is not being used here so this
+is simply a styling example.</p>
+{% set example %}
+
+{{  popover_popperjs(properties={
+    id: _id7,
+    control: sample_control_input(properties={id: _id7}),
+    body: popover_content()
+  }, modifier={ block: ['no-arrow', 'full-width']})
+}}
+
+{% endset %}
+{{ format(example) }}
+
```

Comment:
I'm not sure we need this example because popper.js-positioned components won't be able to have their width set with just CSS since they will exist in a different DOM node (child of body).